ci/CMake/Platforms/toolchain-mingw.cmake

```diff
@@ -0,0 +1,45 @@
+# the name of the target operating system
+SET(CMAKE_SYSTEM_NAME Windows)
+
+# Choose an appropriate compiler prefix
+
+# for classical mingw32
+# see http://www.mingw.org/
+#set(COMPILER_PREFIX "i586-mingw32msvc")
+
+# for 32 or 64 bits mingw-w64
+# see http://mingw-w64.sourceforge.net/
```

Comment:
Please use `https://mingw-w64.sourceforge.io/` to stay aligned with existing SourceForge links.